src/bulkDataAccess.ts

```diff
@@ -16,6 +18,9 @@ export interface InitiateExportRequest {
     type?: string;
```

Comment:
Should this `type` be a string[] i.e. a request can request many types right?

May want to ask Nestor on this one

Reply:
`type` will be a comma separated string when there're multiple types. It's parsed from URL from customer directly and will be split up to an array inside the Glue job.